integration-tests/tests/link-to-new-workspace.js

```diff
@@ -1,16 +1,16 @@
-const { checkbox, click, clickable, clickTableCell, fillIn, waitForNoSpinners } = require('../utils/integration-utils')
+const { checkbox, click, clickable, clickTableCell, fillIn, findHeading, findText, heading, waitForNoSpinners } = require('../utils/integration-utils')
```

Comment:
Are `findHeading` and `heading` used?

Reply:
I'll take those out! They're not needed anymore